packages/web-react/src/components/RadioField/__tests__/RadioField.test.tsx

```diff
@@ -33,4 +33,11 @@ describe('RadioField', () => {
     const element = dom.container.querySelector('input') as HTMLElement;
     expect(element).toHaveClass('RadioField__input');
   });
+
+  it('should have item classname', () => {
```

Comment:
Creating a prop test for this prop like it is in RUI or something like `restPropsTest` will be a better solution for both cases.